packages/codership/mysql-galera/overlay/app-readme.md

```diff
@@ -0,0 +1,13 @@
+# MySQL Galera from Codership
+
+This is a Helm chart for MySQL Galera from Codership.
+
+## Installation
+  - Plan your cluster capacity. Default is 3 nodes, each with 1 vCPU and 4GB of RAM, the storage is 2GB. It is minimal requirement for Galera to work.
+  - Specify a new namespace for the MySQL Galera cluster, like `mysql-galera`.
+  - Install the MySQL Galera chart.
```

Comment:
Looks like it's still a code block. The markdown library that the Rancher UI uses must be picky. I'd remove all spaces before the `-` characters.

![image](https://github.com/user-attachments/assets/c41686c1-4467-4b19-996b-bda0cfab0fbc)